### ISSUE TITLE
How to search the best temperature and alpha

### ISSUE THREAD

**Issue by TimeBear** [yuanli2333/Teacher-free-Knowledge-Distillation#16]
hello ,author.
I read the paper, the parameters (temperature and alpha) are obtained by gird search.
can you release the code . I want to learn it . Thank you.

**Comment by yuanli2333** [comment]
> hello ,author.
> I read the paper, the parameters (temperature and alpha) are obtained by gird search.
> can you release the code . I want to learn it . Thank you.

Hi
It is very simple to search hyper-parameters (grid search), just write a simple python script or bash script to run the main.py and pass the hyper-parameters to args . I will upload a simple example.

**Comment by TimeBear** [comment]
Hello, author.
I am waiting for your simple example.
I really need it.
Thank you very much.

**Comment by JiyueWang** [comment]
Just change the hyperparameter manually

**Comment by yuanli2333** [comment]
> Hello, author.
> I am waiting for your simple example.
> I really need it.
> Thank you very much.

Here is a simple example for how to search hyperparameters: https://github.com/peterliht/knowledge-distillation-pytorch/blob/master/search_hyperparams.py
But make sure you are searching on a validation set that is splitted from the training set rather than the testing set.

**Comment by TimeBear** [comment]
Thankyou
